commit 812d5718c489086425f034177d0b10292f68aadd
Author: Bartosz Markowski <bartosz.markowski@tieto.com>
Date:   Thu Dec 15 11:23:22 2016 +0200

    ath10k: override CE5 config for QCA9377
    
    commit b08b5b53a1ed2bd7a883f8fd29232c8f03604671 upstream.
    
    Similarly to QCA6174, QCA9377 requires the CE5 configuration to be
    available for other feature. Use the ath10k_pci_override_ce_config()
    for it as well.
    
    This is required for TF2.0 firmware. Previous FW revisions were
    working fine without this patch.
    
    Fixes: a70587b3389a ("ath10k: configure copy engine 5 for HTT messages")
    Signed-off-by: Bartosz Markowski <bartosz.markowski@tieto.com>
    Signed-off-by: Kalle Valo <kvalo@qca.qualcomm.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/net/wireless/ath/ath10k/pci.c b/drivers/net/wireless/ath/ath10k/pci.c
index 6063cf4..410bcda 100644
--- a/drivers/net/wireless/ath/ath10k/pci.c
+++ b/drivers/net/wireless/ath/ath10k/pci.c
@@ -3133,7 +3133,7 @@ int ath10k_pci_setup_resource(struct ath10k *ar)
 	setup_timer(&ar_pci->rx_post_retry, ath10k_pci_rx_replenish_retry,
 		    (unsigned long)ar);
 
-	if (QCA_REV_6174(ar))
+	if (QCA_REV_6174(ar) || QCA_REV_9377(ar))
 		ath10k_pci_override_ce_config(ar);
 
 	ret = ath10k_pci_alloc_pipes(ar);
